fvm/transactionEnv.go

```diff
@@ -238,6 +238,15 @@ func (e *TransactionEnv) ValueExists(owner, key []byte) (exists bool, err error)
 	return len(v) > 0, nil
 }
 
+// AllocateStorageAddress allocates new storage address under the owner accounts to store a new register
+func (e *TransactionEnv) AllocateStorageAddress(owner []byte) (uint64, error) {
```

Comment:
`owner` should probably just be a `flow.Address`?

Reply:
updated